lib/components/src/controls/Color.tsx

```diff
@@ -303,9 +303,10 @@ export const ColorControl: FC<ColorProps> = ({
             />
             {presets.length > 0 && (
               <Swatches>
-                {presets.map((preset) => (
+                {presets.map((preset, index: number) => (
                   <WithTooltip
-                    key={preset.value}
+                    // eslint-disable-next-line react/no-array-index-key
+                    key={`${preset.value}-${index}`}
```

Comment:
I don't think there is a way to get unique keys without using the index.

```ts
type ParsedColor = {
  valid: boolean;
  value: string;
  keyword: string;
  colorSpace: ColorSpace;
  [ColorSpace.RGB]: string;
  [ColorSpace.HSL]: string;
  [ColorSpace.HEX]: string;
};
```
Is there a better to solve this? 🤔

Reply:
I think this is fine. The "optimized" way would be to only de-dupe when there are actual duplicates, so `a,b,c,b` might become `a,b1,c,b2`, but I don't think it's worth the effort